### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by mhnazary**
In this project I did:

- [ ] Added visits, vet, specializations tables ✅.
- [ ] Inserted data into them✅✔️.
- [ ]  Link tables.

image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -136,4 +136,84 @@ FROM onwers o
...
+-- 6- Who was Maisy Smith's first visit?
+SELECT vet.name AS vet_name, MIN(v.visit_date) AS first_visit_date
+FROM visits v
+JOIN vets vet ON v.vet_id = vet.id
+JOIN animals a ON v.animal_id = a.id
+WHERE a.owner_id = (SELECT id FROM onwers WHERE full_name = 'Maisy Smith')
+GROUP BY vet.name
+ORDER BY first_visit_date
+LIMIT 1;
+
```
Comment (Strangeal) [inline queries.sql]: - [ ] Great work on your progress in writing queries based on the project requirements! 👏 However, I'd like to provide some feedback to help you improve the accuracy of your results.

  - Upon reviewing the data you've inserted into the `visits` table, it's evident that `Maisy Smith's` first visit was recorded as `boarmon` when sorted by date. This should be reflected in your query's output. Unfortunately, your current query doesn't return `boarmon` as `Maisy Smith's` first visit.

  - To ensure the correctness of your query, please take another look at it and make the necessary adjustments to obtain an accurate result. Additionally, don't forget to update the `screenshot` of the query result in your pull request summary to reflect the corrected outcome.

Happy coding! 😊🚀🍀

File: queries.sql [hunk 2]
```diff
@@ -136,4 +136,84 @@ FROM onwers o
...
+-- 9- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+SELECT s.name AS specialty, COUNT(*) AS visit_count
+FROM visits v
+JOIN animals a ON v.animal_id = a.id
+JOIN specializations sp ON a.species_id = sp.species_id
+JOIN species s ON sp.species_id = s.id
+WHERE a.owner_id = (SELECT id FROM owners WHERE full_name = 'Maisy Smith')
+GROUP BY s.name
+ORDER BY visit_count DESC
```
Comment (Strangeal) [inline queries.sql]: - [ ] Impressive work on your progress! 🌟 However, I'd like to highlight that the project requirements clearly state that you should include `screenshots` of the results of running your database queries. While you've included some `screenshots`, it appears that the `screenshots` for the last two queries are missing from the pull request summary. To ensure the accuracy and completeness of your work, please consider adding these two `screenshots`. 

Happy coding! 😊🚀🍀

**Review by Strangeal — APPROVED** [review]
Hi @mhnazary, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅  
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (mhnazary) [reply queries.sql]: > * Upon reviewing the data you've inserted into the `visits` table, it's evident that `Maisy Smith's` first visit was recorded as `boarmon` when sorted by date. This should be reflected in your query's output. Unfortunately, your current query doesn't return `boarmon` as `Maisy Smith's` first visit.

**Review by Davidosky007 — APPROVED** [review]
Hi @mhnazary 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Well-written PR 👍 
* All tables joined properly 👍 
* Good looking readme 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**